keras/models.py

```diff
@@ -370,4 +379,33 @@ def load_weights(self, filepath):
             g = f['layer_{}'.format(k)]
             weights = [g['param_{}'.format(p)] for p in range(g.attrs['nb_params'])]
             self.layers[k].set_weights(weights)
-        f.close()
\ No newline at end of file
+        f.close()
+
+
+class Autoencoder(Sequential):
```

Comment:
What is this model doing in this PR, given that autoencoders are being implemented with a layer in `core.py`...?